draft-duke-quic-load-balancers.md

```diff
@@ -302,14 +357,18 @@ and server.
 For Plaintext CID Routing, this consists of the Routing Bits, Divisor, and
 Modulus. The Modulus is unique to each server, but the others MUST be global.
 
-For Encrypted CID Routing, this consists of the Server ID, Server ID Length,
-Key, and Connection ID Length.  The Server ID is unique to each server, but the
+For Stream Cipher CID Routing, this consists of the Server ID, Server ID Length,
+Key, and Nonce Length.  The Server ID is unique to each server, but the
```

Comment:
Maybe you need token as well, because we have constructions like: "The server pads its nonce to 16 octets using the earliest octets of the token, and uses the result as the counter input to AES-CTR"?